src/utils.py

```diff
@@ -10,6 +10,9 @@ def create_line_plot(df, x_column, y_columns, title, legend_names):
     for original_col, new_name in zip(y_columns, legend_names):
         plot_df[new_name] = plot_df[original_col]
 
+    config = {
+        "displayModeBar": False,
+    }
```

Comment:
You can remove this, since it's set in the dcc.Graph object which wraps the plotly plot